### PR TITLE
fix(css): declaration comments

### PR DESCRIPTION
**Pull request by evilebottnawi**
Now we don't eat comments. We can merge this as is and in next PR discussion about place comments or do this here :+1: Welcome to feedback!

### CODE REVIEW

File: tests/css_comments/declaration.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+a {
+    /* comment 1 */
+    /* comment 2 */ padding /* comment 3 */ : /* comment 4 */ 10px /* comments 5 */ 10px /* comment 6 */; /* comment 7 */
```
Comment (j-f1) [inline tests/css_comments/declaration.css]: `comment 5`?

Comment (evilebottnawi) [reply tests/css_comments/declaration.css]: @j-f1 fixed :+1:

**Comment by evilebottnawi** [comment]
Also tomorrow fix `!default` and `!global`, wait feedback about better place for each comment

**Comment by evilebottnawi** [comment]
@j-f1 @lydell seems all comments exists and don't eaten, merge this as is (and open issue about position) or fix places here?

**Comment by evilebottnawi** [comment]
Merge as is. Issue: https://github.com/prettier/prettier/issues/3905

**Comment by Kerrick** [comment]
Does this also fix #2686?

**Comment by j-f1** [comment]
Nope, the issue is still present in this PR’s deploy preview.

**Comment by evilebottnawi** [comment]
@Kerrick no, WIP right now

**Pull request merged**